app/templates/components/notification-dropdown.hbs

```diff
@@ -10,20 +10,22 @@
       {{/link-to}}
     </div>
     <div class="ui basic right aligned segment weight-400">
-      <a href="#">{{t 'Mark all as Read'}}</a>
+      <a href="#" {{action 'markAllRead'}}>{{t 'Mark all as Read'}}</a>
     </div>
   </div>
   <div class="ui fluid link celled selection list">
     {{#each notifications as |notification|}}
```

Comment:
Now read and unread notifications look the same. Ensure there is a subtle difference.